### PR TITLE
feat(infracijioagents1): add new cluster in kubernetes-management

### PR DESCRIPTION
**Pull request by smerle33**
as per 
- https://github.com/jenkins-infra/helpdesk/issues/3923

adding the new cluster in the kubernetes-management with the secrets in charts-secrets for infra.ci -- commit [2fcc66d]

need 
- #5313

### CODE REVIEW

File: config/jenkins-kubernetes-agents_infra.ci.jenkins.io.yaml [hunk 1]
```diff
@@ -1 +1,2 @@
-existingServiceAccount: jenkins-infra:jenkins-infra-controller
+quotas:
+  pods: 150
```
Comment (smerle33) [inline config/jenkins-kubernetes-agents_infra.ci.jenkins.io.yaml:2]: need to confirm this number

Comment (smerle33) [reply config/jenkins-kubernetes-agents_infra.ci.jenkins.io.yaml:2]: I will change this in a PR if needed

**Review by dduportal — APPROVED** [review]
- Helmfile diff is good 👍 Credentials and routing are ✅ 
- Code looks good!

**Comment by dduportal** [comment]
For info, this PR required a fixup: https://github.com/jenkins-infra/kubernetes-management/pull/5318

**Pull request merged**